server.js

```diff
@@ -0,0 +1,18 @@
+var path = require('path');
+var express = require('express');
+var app = express();
+var PORT = process.env.PORT || 8080
+
+app.use(express.static(path.join(__dirname, 'dist')));
+
+app.get('/', function(request, response) {
+  response.sendFile(__dirname + '/dist/index.html')
+});
+
+app.listen(PORT, function(error) {
+  if (error) {
```

Comment:
since just simple `if` `else` can use ternary?

```
error
  ? console.error(error)
  :  console.info(`==> 🌎 Listening on port ${PORT}. Visit http://localhost:${PORT}/ in your browser.`)
```